src/tests/java/picard/util/IlluminaUtilTest.java

```diff
@@ -73,15 +73,4 @@
             }
         };
     }
-
-    /*
-    @Test(dataProvider = "solexaQualStrToPhreds")
```

Comment:
this has been commented out since it was originally written....I have no idea why.